relay/kusama/src/lib.rs

```diff
@@ -1247,6 +1249,12 @@ impl InstanceFilter<RuntimeCall> for ProxyType {
 				RuntimeCall::System(frame_system::Call::remark { .. }) |
 					RuntimeCall::System(frame_system::Call::remark_with_event { .. })
 			),
+			ProxyType::ParaRegistration => matches!(
+				c,
+				RuntimeCall::Registrar(paras_registrar::Call::reserve { .. }) |
```

Comment:
You don't need batch or similar as well?

Reply:
And register batched with remove proxy?

Reply:
Not being used right now, but I still think it could be useful down the line. I just added the Utilities pallet as well for this proxy type.